src/core/libmaven/mzUtils.cpp

```diff
@@ -55,7 +55,13 @@ namespace mzUtils {
     vector<string> split(const string& str, const string& sep)
     {
         vector<string> result;
-        boost::split(result, str, boost::is_any_of(sep));
+        int start = 0;
+        int end = 0;
+        while(end != string::npos){
+            end = str.find(sep, start);
+            result.push_back(str.substr(start, end - start));
+            start = end + sep.length();
+        }
```

Comment:
Is this fix for reading MoNa library properly?

Reply:
Yes. The split method was splitting on every character within the string individually - not the string as a whole.

Reply:
cool